src/fides/api/ops/api/v1/endpoints/user_endpoints.py

```diff
@@ -191,3 +211,120 @@ def user_logout(
     logger.info("Logging out user.")
     if client:
         client.delete(db)
+
+
+@router.put(
+    urls.SYSTEM_MANAGER,
+    dependencies=[Security(verify_oauth_client, scopes=[SYSTEM_MANAGER_UPDATE])],
+    response_model=List[SystemSchema],
+)
+def update_system_manager(
+    *,
+    db: Session = Depends(deps.get_db),
+    user_id: str,
+    systems: List[FidesKey],
+) -> List[SystemSchema]:
```

Comment:
I'm returning a detailed representation of the system here.  Would there be any reasons to limit this and just return a subset of the system fields here?

Reply:
that seems okay to me!